Sources/Purchasing/Purchases/PurchasesType.swift

```diff
@@ -941,3 +941,18 @@ public protocol PurchasesSwiftType: AnyObject {
     #endif
 
 }
+
+// MARK: -
+
+/// Interface for ``Purchases``'s internal-only methods.
+internal protocol InternalPurchasesType: AnyObject {
+
+    /// Performs an unauthenticated request to the API to verify connectivity.
+    /// - Throws: `PublicError` if request failed.
+    @available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)
+    func healthRequest() async throws
+
+    @available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)
+    func offerings(fetchPolicy: OfferingsManager.FetchPolicy) async throws -> Offerings
```

Comment:
Why is `offerings` considered internal?

Reply:
This is just an overload that takes `OfferingsManager.FetchPolicy`, which is internal. It exposes the feature added in #1976. The method without parameters is still part of the `public PurchasesType protocol`.